src/components/TextBackground.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from "react";
+
+interface IProps {
+  width: number;
+  height: number;
+  deg: number;
+  left: number;
+  bottom: number;
+  right: number;
+  top: number;
+  zIndex: number;
+}
+
+export const TextBackground: FC<IProps> = ({
+  width,
+  height,
+  deg,
+  right,
+  left,
+  zIndex = 0,
+  bottom,
+  top,
+  children
+}) => (
+  <div style={{ position: "relative" }}>
```

Comment:
https://tailwindcss.com/docs/position/#app we have a tailwind class for relative position